build.gradle

```diff
@@ -137,6 +137,8 @@ dependencies {
     testImplementation 'com.github.tomakehurst:wiremock-standalone:2.26.0'
     testImplementation 'io.rest-assured:rest-assured:3.3.0'
     testImplementation 'io.rest-assured:spring-mock-mvc:3.3.0'
+    testImplementation 'com.nhaarman:mockito-kotlin-kt1.1:1.6.0'
```

Comment:
This looked weird, so I googled it.  Found out that we're using the old version, should be
``` 
com.nhaarman.mockitokotlin2:mockito-kotlin:2.2.0
```
instead.  No idea of the difference, but will be investigating today!

Reply:
BTW just had to change package name and all tests passed